src/screens/BenefitManipulationScreen.tsx

```diff
@@ -1,12 +1,65 @@
-import React from 'react';
-import { StyleSheet, Text, View, StatusBar } from 'react-native';
+import React, { useState } from 'react';
+import { StyleSheet, Text, View, StatusBar, Platform } from 'react-native';
+import Icon from 'react-native-vector-icons/MaterialCommunityIcons';
 
+import TopBar from '../components/TopBar';
+import InputField from '../components/InputField';
+import FileDropZone from '../components/FileDropZone';
+import DatePicker from '../components/DatePicker';
+import CustomButton from '../components/CustomButton';
+import TapBar from '../components/TapBar';
+import BoxContainer from '../components/BoxContainer';
+
+/*
+todo: 
+      fix inputfields to not fuck up the screen when inputing
+      integrate/implement datepicker
+      check if existing or new benefit
+      implementation for new below
+      for existing same but info given eg. useState(card prop)
+      ^de facto change only button onPress?
+      button onPress sends request through api
```

Comment:
Tak, API powinno to poprawnie handlować. Card prop nie trzeba trzymać w useState, bo to co przekażesz jako propsa (i będziesz na tym wykonywał obliczenia) przy zmianie propsa będzie rerenderowało wszystko inne.